JOB_INSIGHT_FAILURE_HISTORY_ANALYSIS_PROMPT.md

```
@@ -0,0 +1,57 @@
+   <!-- To be used with https://github.com/myk-org/rootcoz
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)